apps/api/src/.env.test

```diff
@@ -2,7 +2,7 @@ GOOGLE_OAUTH_CLIENT_ID=11
 GOOGLE_OAUTH_CLIENT_SECRET=11
 GOOGLE_OAUTH_REDIRECT=http://localhost:3000/v1/auth/google/callback
 STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
-BLUEPRINT_CREATOR="<BLUEPRINT_CREATOR>"
+BLUEPRINT_CREATOR=645b648b36dd6d25f8650d37
```

Comment:
This is the part of the trick... The `BLUEPRINT_CREATOR` is the `organizationId` of the org that has the blueprints (aka. notification-templates) on the prod env. In order for blueprints API to work in the testing environment I've set this hardcoded `organizationId` value for API and Cypress config as well. Now the API knows where it should find the blueprints. And in Cypress, I've prepared the plugin code that creates the org with that `id`, prod env, and blueprints, so the API can return something for tests.